lib/pizza_bot.rb

```diff
@@ -1,4 +1,5 @@
 class PizzaBot
+  # First round of edits
   attr_reader :width, :height, :max_dimensions, :coordinates, :directions
 
   def initialize(string)
```

Comment:
Calculate Directions Method: "The grand daddy of them all". Again i'm doing some janky string parsing on lines 21-23 which I don't love. Method is also obviously too long and not abiding by SRP so lmk if you have any good ideas of how to break this up/optimize.